modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/IBlogPostRepository.cs

```diff
@@ -44,4 +44,6 @@ Task<List<CmsUser>> GetAuthorsHasBlogPostsAsync(
     Task<CmsUser> GetAuthorHasBlogPostAsync(Guid id, CancellationToken cancellationToken = default);
 
     Task<bool> HasBlogPostWaitingForReviewAsync(CancellationToken cancellationToken = default);
+
+    Task UpdateBlogAsync(Guid sourceBlogId, Guid? targetBlogId, CancellationToken cancellationToken = default);
```

Comment:
It would be better to set the nullable parameter as null as default, so we don't explicitly pass it.

```suggestion
    Task UpdateBlogAsync(Guid sourceBlogId, Guid? targetBlogId = null, CancellationToken cancellationToken = default);
```